src/pages/cli/project/troubleshooting.mdx

```diff
@@ -209,6 +209,29 @@ If `amplify push` fails while deploying any of the scenarios below, refer to the
 1.	[Backfill OpenSearch index from DynamoDB table](/cli/graphql/troubleshooting/#deploying-multiple-index-changes-at-once)
 1.	[Index with Multi-Sort Key Fields](/cli/graphql/troubleshooting/#index-with-multiple-sort-key-fields)
 
+## Troubleshooting "amplify pull" failures
+
+### Pull failures with large number of UI components
+<Callout information>
+
+**Errors:**
+
+"⠋ Generating UI components...🛑 Codegen job never succeeded before timeout"
+"✖ Failed to sync UI components"
```

Comment:
can you put this in two separate fenced code blocks? This will help present the errors how they appear in the terminal with a monospace font.

Reply:
This format was to keep the document consistent as previous errors on the same page had similar styles